src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -259,4 +263,31 @@ default GameModeData getGameModeData() {
      */
     void setSleepingIgnored(boolean sleepingIgnored);
 
+    /**
+     * Gets a collection of {@link ServerBossBar}s this player is a member of.
+     *
+     * <p>The returned collection is unmodifiable.</p>
+     *
+     * @return The boss bars
+     */
+    Collection<ServerBossBar> getBossBars();
```

Comment:
It sounds like from the methods below that it's `ServerBossBar`s that have a list of players (and thusly an `M:N` relationship), but everything goes through `ServerBossBar`. So my question I guess is, where are boss bars stored if not otherwise on players? Does the `Server` keep track of them? Does the `World`?